### ISSUE TITLE
Clarification on EMDiffuse baseline evaluation and significant discrepancy with concurrent CVPR 2026 work (EMGauss)

### ISSUE THREAD

**Issue by hmzawz2** [I2-Multimedia-Lab/VEMamba#3]
Hi VEMamba Authors,

Congratulations on the acceptance of your excellent work at CVPR 2026! Adapting the Mamba architecture to volume electron microscopy (vEM) is a very inspiring and efficient solution for the community.

While extensively reading and comparing concurrent CVPR 2026 papers on vEM isotropic reconstruction, we noticed a highly confusing numerical discrepancy regarding the baseline evaluations (specifically EMDiffuse). We have attached a summary table for clarity (see the image below).

Could you kindly share your insights on the following two points?

1. SOTA Diffusion Model vs. Simple Interpolation Baseline
In your evaluation on the EPFL dataset (Table 1), under the highly challenging 8x scale factor (with complex degradation including blur, downsampling, and noise), the mathematical Baseline (interpolation) achieves a PSNR of 25.558. However, EMDiffuse—a heavy, state-of-the-art generative model—only achieves 24.947 under the exact same setting.
Could you elaborate on why a powerful diffusion model is completely outperformed by a training-free mathematical interpolation? Does EMDiffuse suffer from severe mode collapse under your specific complex degradation simulation, or is the interpolation inherently favored by this specific evaluation protocol?

2. The "Difficulty Inversion" Discrepancy with EMGauss (CVPR 2026)
Interestingly, another concurrent CVPR 2026 paper, EMGauss (arXiv:2512.06684), evaluated EMDiffuse on the exact same EPFL dataset. In their setting, which uses a much simpler 6x manual downsampling without additional complex noise/blur, EMDiffuse only achieves a PSNR of 23.34.
It is fascinating and counter-intuitive that EMDiffuse performs significantly better on your harder 8x complex degradation task (24.947) than on EMGauss's easier 6x downsampling task (23.34).

Did you retrain EMDiffuse using a different strategy, or do you believe this massive discrepancy across two concurrent CVPR 2026 papers stems from differences in cross-volume generalization vs. target-volume overfitting?

<img width="1095" height="790" alt="Image" src="https://github.com/user-attachments/assets/d622abe9-082b-4f70-90c8-f94c5029a870" />

Understanding these baseline dynamics would greatly help the vEM computation community standardize evaluation protocols. Thank you again for your time and contribution!

**Comment by xiaogaogao26** [comment]
Hi, thanks for the thoughtful question.

Regarding EMDiffuse, we noticed that several key components are missing in the official repository, so we reimplemented the method according to the paper and repeated the experiments multiple times. In our reproduction, EMDiffuse indeed performs slightly worse than the training-free interpolation baseline under our complex degradation simulation (blur + downsampling + noise). One possible reason is that diffusion models may struggle under such degradation settings, potentially leading to unstable reconstruction behavior. However, this is an empirical observation from our reproduction rather than a definitive conclusion.

PSNR, SSIM, and LPIPS are standard evaluation metrics and do not inherently favor any particular method.

As for the discrepancy with results reported in other papers, it may stem from differences in implementation details, input volume sizes, or evaluation protocols (e.g., slice-wise vs. volume-wise metric computation). In practice, comparisons within the same paper under a consistent protocol are often more reliable than direct comparisons across different works.
